scala-package/pom.xml

```diff
@@ -423,23 +590,11 @@
       <version>1.7.7</version>
       <scope>provided</scope>
     </dependency>
-    <dependency>
-      <groupId>org.scalatest</groupId>
-      <artifactId>scalatest_2.11</artifactId>
```

Comment:
Can you try to keep the dependencies in the dependency section so it is easier to keep track of them. For scalatest, you could just use `<artifactId>scalatest_${scala.binary.version}</artifactId>`. If you need to vary the version, you could use a property.

Reply:
OK. But `scala-parallel-collections` in `core/pom.xml` will remain in profile dependencies because in 2.11 and 2.12 it is part of standard library.

Reply:
I tried to do this. But it doesn't working. Looks like `maven-remote-resources-plugin` doesn't understand properties in dependencies. Here is error log https://gist.github.com/fomkin/a26aefc59eaca1148dfed3719a8144f0

Reply:
Well. It solves by removing _default_ properties: `maven-remote-resources-plugin` ignores the property from a profile if it has default values.